pyproject.toml

```diff
@@ -43,17 +44,19 @@ docs = [
     "sphinx-automodapi",
     "sphinx-rtd-theme",
     "stsci-rtd-theme",
+    "graphviz",
 ]
 test = [
-    "pytest>=7.0",
+    "pytest>=6.0",
+    "pytest-cov",
 ]
 
 [build-system]
 requires = [
-    "setuptools>=61.2",
-    "setuptools_scm[toml] >=6.2",
+    "setuptools>=30.3",
+    "setuptools_scm[toml]>=3.4",
     "wheel",
-    "numpy>=1.25,<2",
+    "oldest-supported-numpy; python_version>='3.7'",
```

Comment:
I am not sure if this is a good idea. See https://github.com/astropy/astropy/issues/14915